include/openenclave/attestation/attester.h

```diff
@@ -73,17 +78,23 @@ oe_result_t oe_attester_select_format(
  *
  * @experimental
  *
- * @param[in] evidence_format The format UUID of the evidence to be generated.
- * @param[in] custom_claims The optional custom claims list.
- * @param[in] custom_claims_length The number of custom claims.
+ * @param[in] format_id The format ID of the evidence to be generated.
+ * @param[in] flags A bit-wise parameter. Currently there is one bit
+ * defined: OE_EVIDENCE_FLAGS_BIT_WRAP_HEADER. If this bit is set,
+ * the evidence and
+ * endorsements will be wrapped with an attestation header containing the
+ * format ID.
```

Comment:
Fix up the line spacing.
```suggestion
 * defined: OE_EVIDENCE_FLAGS_BIT_WRAP_HEADER. If this bit is set,
 * the evidence and endorsements will be wrapped with a header
 * containing the format ID.
```

Reply:
Thanks!